wpilibc/src/main/native/include/frc/IterativeRobotBase.h

```diff
@@ -63,10 +63,8 @@ class IterativeRobotBase : public RobotBase {
    * which will be called when the robot is first powered on. It will be called
    * exactly one time.
    *
-   * Warning: the Driver Station "Robot Code" light and FMS "Robot Ready"
-   * indicators will be off until RobotInit() exits. Code in RobotInit() that
-   * waits for enable will cause the robot to never indicate that the code is
-   * ready, causing the robot to be bypassed in a match.
+   * Note: This method is functionally identical to the class constructor so
+   * that should be used instead.
    */
```

Comment:
(making a thread so we can keep the following discussion in one place)

We might want to think about the implications for RobotPy here. IIRC we've historically recommended using `robotInit` in Python because teams can easily forget to call `super().__init__()` when overriding `__init__`.

Reply:
That's really weird. C++ and Java both call the default constructor for you, but Python doesn't: https://godbolt.org/z/W51a716qz